src/app/signup/page.jsx

```diff
@@ -2,11 +2,11 @@
 
 import { useState } from "react";
```

Comment:
separate this page into a react server component and client component.
-> signup
-  page.tsx -> contains your react server components
- signup.client.tsx -> contains your login functionality, and uses `use client`